drizzlepac/haputils/poller_utils.py

```diff
@@ -323,22 +338,26 @@ def build_mvm_tree(obset_table):
         filt = determine_filter_name(orig_filt)
         row['filters'] = filt
         layer = (skycell, filt, exp_layer, year_layer)
+        if exp_layer in ["ALL", 'all', 'None', '', ' ', None]:
```

Comment:
In this general area it would be good to have some documentation.

Reply:
Resolved in 21fa202.  Added several more doc strings to this section of the code.